extension/upnp/upnp.js

```diff
@@ -60,38 +60,39 @@ module.exports = class {
             instance = this;
             this.refreshTimers = {};
             
-            upnpIntervalHandler = setInterval(
-                () => {
-                    log.info("UPnP periodical check starts")
-                    if (upnpMappings.isEmpty) {
-                        log.info("No mapping registered.")
-                        return;
-                    }
-                    upnpClient.getMappings((err, results) => {
-                        if (err) {
-                            log.error("Failed to get current mappings", err);
-                            return;
-                        }
-                        log.info("Current mappings: ", results);
-
-                        upnpMappings.forEach((check) => {
-                            log.info("Checking registered mapping:", check);
-                            if (_.isEmpty(
-                                results.find((m) => mappingCompare(m, check))
-                            )) {
-                                log.info("Mapping no longer exists, adding back to router...")
-                                let { protocol, localPort, externalPort, description } = check;
-                                this.addPortMappingUPNP(protocol, localPort, externalPort, description)
-                            } else {
-                                log.info("Mapping still exists")
-                            }
-                        })
-                    })
-                },
-                upnpCheckInterval
-            )
+            if (process.title === "FireMain") {
```

Comment:
use this please :)

https://github.com/firewalla/firewalla/blob/master/net2/Firewalla.js#L279